packages/request-client.js/test/http-request-network.test.ts

```diff
@@ -65,6 +65,9 @@ describe('HttpRequestNetwork', () => {
           r(e);
         });
       });
+
+      console.log('Error :', error.message);
+
```

Comment:
Agreed :+1: This will be removed before merging. 
@aimensahnoun consider adding `FIXME: remove before merging` when you add temporary debug code like this.